app/repos/people.js

```diff
@@ -109,7 +185,7 @@ const getPersonAndDogsByReference = async (reference, transaction) => {
         {
           model: sequelize.models.person_contact,
           as: 'person_contacts',
-          separate: true, // workaround to prevent 'contact_type_id' being truncated to 'contact_type_i'
```

Comment:
I would argue this is quite an important comment to keep in, explaining why the 'separate: true' is used. Did you mean to remove?

Reply:
Comments like that are better off to have in documentation. Can be a bit messy keeping inline comments like that in in my opinion.

Reply:
Ok, but do we have any documentation yet?